app/assets/javascripts/short-form/ShortFormNavigationService.js.coffee

```diff
@@ -411,12 +415,12 @@ ShortFormNavigationService = (
     else
       'household-intro'
 
-  Service.getPrevPageOfIncomeSection = ->
+  Service.getPrevPageOfIncomePage = ->
     listing = ShortFormApplicationService.listing
     if ListingIdentityService.isSale(listing)
-      Service.getNextReservedPageIfAvailable(RESERVED_TYPES.DISABLED, 'prev')
+      Service.getNextReservedPageIfAvailable(Service.RESERVED_TYPES.DISABLED, 'prev')
```

Comment:
interesting, so we're getting rid of back-to-back pages.

Reply:
yeah, in retrospect perhaps we should have made these two one story